lib/main.coffee

```diff
@@ -101,8 +101,24 @@ module.exports =
             return [] unless result.files[filePath]
             messages = result.files[filePath].messages
           return messages.map (message) =>
-            startPoint = [message.line - 1, message.column - 1]
-            endPoint = [message.line - 1, message.column]
+
+            # fix column in line with tabs
+            column = message.column
+            if @tabWidth > 1
```

Comment:
if `@tabwidth = 1` than the character column will be same as the screen column so there is no need to calculate it.

The column returned by PHPCS is only incorrect when `@tabwidth > 1`